internals-js/src/__tests__/coreSpec.test.ts

```diff
@@ -209,4 +209,74 @@ describe('removeAllCoreFeatures', () => {
     expect(schema.elementByCoordinate("@foo__qaz")).toBeUndefined();
     expect(schema.elementByCoordinate("@foo__quz")).toBeUndefined();
   });
+
+  it('does not remove tags in API schema', () => {
+    const subgraph = buildSubgraph('S', '', gql`
+      extend schema
+        @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@tag"])
+
+      type Query {
+        q: Int
+      }
+
+      type User {
+        k: ID
+        a: Int @tag(name: "foo")
+      }
+    `);
+    const { schema } = subgraph;
+    removeAllCoreFeatures(schema, ["https://specs.apollo.dev/federation/v2.0#@tag"]);
+    schema.validate();
+
+    expect(schema.elementByCoordinate("@tag")).toBeDefined();
+  });
+
+  it('removing in one schema should not affect another', () => {
+    const subgraph = buildSubgraph('S', '', gql`
+      extend schema
+        @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@tag"])
+        @link(url: "https://specs.apollo.dev/link/v1.0")
+
+      type Query {
+        q: Int
+      }
+
+      type User {
+        k: ID
+        a: Int @tag(name: "foo")
+      }
+    `);
+    const { schema } = subgraph;
+    removeAllCoreFeatures(schema, ["https://specs.apollo.dev/link/v1.0#@tag"]);
```

Comment:
I think this should be "https://specs.apollo.dev/link/v1.0#@link"?

Reply:
I think it's specifically testing that the v1 tag doesn't remove the v2 tag that gets put in the supergraph (the previous test)